Source/Fuse.Common/Internal/MiniList.uno

```diff
@@ -11,16 +20,26 @@ namespace Fuse.Internal
 	struct MiniList<T> : IList<T> where T : class
 	{
 		object _list;
+		ObjectList<T> AsList { get { return (ObjectList<T>)_list; } }
+		
+		MiniListMode _mode = MiniListMode.Empty;
```

Comment:
Right, you're right, going back to the old discussion, this was about allocations, not size. But that makes the type confusingly named, and that gets slightly worse with this change. But that's not a functional problem, so maybe we can deal with that at some later point instead.